bundles/com.espressif.idf.serial.monitor/META-INF/MANIFEST.MF

```diff
@@ -33,4 +34,6 @@ Bundle-ClassPath: lib/javax.websocket-client-api-1.1.jar,
  lib/jsr305-3.0.2.jar,
  lib/log4j-1.2.14.jar,
  lib/pty4j-0.8.6.jar,
- lib/purejavacomm-0.0.11.1.jar
+ lib/purejavacomm-0.0.11.1.jar,
+ lib/Java-WebSocket-1.5.2.jar,
+ lib/slf4j.api-1.6.1.jar
```

Comment:
Are we using this? if not could we remove slf4j references

Reply:
slf4j is required for the java-websocket jar we are using without it the lib doesn't work